cli/tests/stake.rs

```diff
@@ -345,7 +363,7 @@ fn test_stake_delegation_force() {
         withdrawer: None,
         withdrawer_signer: None,
         lockup: Lockup::default(),
-        amount: SpendAmount::Some(50_000_000_000),
+        amount: SpendAmount::Some(25_000_000_000),
```

Comment:
To confirm, this changed because you're creating a second stake account now and you don't want to exhaust the `100_000_000_000` airdop?